### DISCUSSION TITLE
HELP!

### DISCUSSION THREAD

**Discussion by GGboogie** [3kh0/ext-remover#588] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Please help me. I am currently being blocked by GoGuardian and Cisco Umbrella which are both annoying and I do not know what to do. This is what Cisco Umbrella and GoGuardian looks like:

![Screenshot 2023-03-27 5 00 04 PM](https://user-images.githubusercontent.com/97118713/228093261-260d53e1-29b9-47be-8f86-76df7aa2c8d7.png)
![Screenshot 2023-03-27 5 05 47 PM](https://user-images.githubusercontent.com/97118713/228093640-0a2df497-9091-43ed-9437-4328b9422fd3.png)

Anyways, my school has blocked ALL of the proxies and I can't find a way around this problem. I do not want to unenroll as I know I will get in trouble but I AM BORED AS HELL IN SCHOOL.

**Comment by Aka-but-nice** [comment]
> Please help me. I am currently being blocked by GoGuardian and Cisco Umbrella which are both annoying and I do not know what to do. This is what Cisco Umbrella and GoGuardian looks like:
> 
> ![Screenshot 2023-03-27 5 00 04 PM](https://user-images.githubusercontent.com/97118713/228093261-260d53e1-29b9-47be-8f86-76df7aa2c8d7.png) ![Screenshot 2023-03-27 5 05 47 PM](https://user-images.githubusercontent.com/97118713/228093640-0a2df497-9091-43ed-9437-4328b9422fd3.png)
> 
> Anyways, my school has blocked ALL of the proxies and I can't find a way around this problem. I do not want to unenroll as I know I will get in trouble but I AM BORED AS HELL IN SCHOOL.

Also, DM me "Aka, but nice#5094"

**Comment by S-PScripts** [comment, marked as answer]
r
